public/locales/en/labdetails.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Some of these might not be necessary for labs, if anyone with a little bit more domain knowledge knows which aren't needed please let me know and I'll remove them.